package.json

```diff
@@ -4,6 +4,7 @@
   "homepage": "https://github.com/marionettejs/backbone.radio",
   "version": "2.0.0",
   "main": "build/backbone.radio.js",
+  "module": "build/backbone.radio.esm.js",
```

Comment:
It's wired into the build so would get generated with the rest of the files in `build/` in the future.  

I'm not sure what needs to be done, if anything for the NPM package to include the ES modules.